diff --git a/layout_builder_iframe_modal.module b/layout_builder_iframe_modal.module
index 0906f752b7940691a8e1ce64117fae99be590b4b..27e88dd2909e6913c802e4a51db2a908e297de3c 100644
--- a/layout_builder_iframe_modal.module
+++ b/layout_builder_iframe_modal.module
@@ -150,5 +150,8 @@ function layout_builder_iframe_modal_form_alter(&$form, FormStateInterface $form
     '#weight' => 16,
     '#title' => t('Rebuild layout'),
     '#url' => $url,
+    '#attributes' => [
+      'class' => ['hidden'],
+    ],
   ];
 }
